scales/definitions/_newscale.json.template

```diff
@@ -22,7 +22,7 @@
             },
             {
                 "interval": 5,
-                "length": 25,
+                "length": 30,
```

Comment:
I'd like to keep the 25 value for the template, because that's the value that was used for creating most of the existing scales. For the scales that use liters this is sufficient for the scales using up to 99l. In my opinion 30 creates to much space for "regular" scales. It is ok for your newly added scales to use a different length if you like it better.